src/utils.cpp

```diff
@@ -19,10 +19,18 @@ void read_moduli_from_csv( \
     mpz_init(n);
     int err = 0;
     bool zero = false;
+    cout << base << endl;
```

Comment:
Put a more verbose comment in line 14 instead of this. Something like 
```Reading moduli from file.csv (base 10)```

Reply:
Done